modules/images-editing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,119 @@
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+Name:                       rosa-hcp-test
+Domain Prefix:              rosa-hcp-test
+Display Name:               rosa-hcp-test
+ID:                         <cluster_hcp_id>
+External ID:                <cluster_hcp_id>
+Control Plane:              ROSA Service Hosted
+OpenShift Version:          4.Y.Z
+Channel Group:              stable
+DNS:                        <dns>
+AWS Account:                <AWS_id>
+AWS Billing Account:        <AWS_id>
+API URL:                    <ocm_api>
+Console URL:
+Region:                     us-east-1
+Availability:
+ - Control Plane:           MultiAZ
+ - Data Plane:              SingleAZ
+
+Nodes:
+ - Compute (desired):       2
+ - Compute (current):       2
+Network:
+ - Type:                    OVNKubernetes
+ - Service CIDR:            <service_cidr>
+ - Machine CIDR:            <machine_cidr>
+ - Pod CIDR:                <pod_cidr>
+ - Host Prefix:             /23
+ - Subnets:                 <subnet_ids>
+EC2 Metadata Http Tokens:   optional
+Role (STS) ARN:             arn:aws:iam::<AWS_id>:role/<account_roles_prefix>-HCP-ROSA-Installer-Role
+Support Role ARN:           arn:aws:iam::<AWS_id>:role/<account_roles_prefix>-HCP-ROSA-Support-Role
+Instance IAM Roles:
+ - Worker:                  arn:aws:iam::<AWS_id>:role/<account_roles_prefix>-HCP-ROSA-Worker-Role
+Operator IAM Roles:
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-kube-system-capa-controller-manager
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-kube-system-control-plane-operator
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-kube-system-kms-provider
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-openshift-image-registry-installer-cloud-cred
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-openshift-ingress-operator-cloud-credentials
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-openshift-cluster-csi-drivers-ebs-cloud-credent
+ - arn:aws:iam::<AWS_id>:role/<operator_roles_prefix>-openshift-cloud-network-config-controller-cloud
+Managed Policies:           Yes
+State:                      ready
+Private:                    No
+Delete Protection:          Disabled
+Created:                    Oct 01 2030 09:48:52 UTC
+User Workload Monitoring:   Enabled
+OIDC Endpoint URL:          https://<endpoint> (Managed)
+Audit Log Forwarding:       Disabled
+External Authentication:    Disabled
+Etcd Encryption:            Disabled
+Registry Configuration:
+ - Allowed Registries:      <allowed_registry> <1>
+ - Insecure Registries:     <insecure_registry> <2>
+ - Allowed Registries for Import: <3>
+    - Domain Name:          <domain_name> <4>
+    - Insecure:             true <5>
+----
+<1> `Allowed Registries`: A comma-separated list of registries for which image pull and push actions are allowed.
```

Comment:
In above edition, it used 'blocked registries',so it should be consistent.

Reply:
Updated line 29 for "allowed" instead of "blocked":
![image](https://github.com/user-attachments/assets/18de7a53-52a9-4817-97b7-5f8f251d3fa6)